x-pack/plugins/synthetics/common/rules/legacy_uptime/translations.ts

```diff
@@ -0,0 +1,160 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+
+export const MonitorStatusTranslations = {
+  defaultActionMessage: i18n.translate(
+    'xpack.synthetics.alerts.monitorStatus.defaultActionMessage',
+    {
+      defaultMessage:
+        'Monitor {monitorName} with url {monitorUrl} from {observerLocation} {statusMessage} The latest error message is {latestErrorMessage}, checked at {checkedAt}',
+      values: {
+        monitorName: '{{context.monitorName}}',
+        monitorUrl: '{{{context.monitorUrl}}}',
+        statusMessage: '{{{context.statusMessage}}}',
+        latestErrorMessage: '{{{context.latestErrorMessage}}}',
+        observerLocation: '{{context.observerLocation}}',
+        checkedAt: '{{context.checkedAt}}',
+      },
+    }
+  ),
+  defaultSubjectMessage: i18n.translate(
+    'xpack.synthetics.alerts.monitorStatus.defaultSubjectMessage',
+    {
+      defaultMessage: 'Monitor {monitorName} with url {monitorUrl} is down',
+      values: {
+        monitorName: '{{context.monitorName}}',
+        monitorUrl: '{{{context.monitorUrl}}}',
+      },
+    }
+  ),
+  defaultRecoverySubjectMessage: i18n.translate(
+    'xpack.synthetics.alerts.monitorStatus.defaultRecoverySubjectMessage',
+    {
+      defaultMessage: 'Monitor {monitorName} with url {monitorUrl} has recovered',
+      values: {
+        monitorName: '{{context.monitorName}}',
+        monitorUrl: '{{{context.monitorUrl}}}',
+      },
+    }
+  ),
+  defaultRecoveryMessage: i18n.translate(
+    'xpack.synthetics.alerts.monitorStatus.defaultRecoveryMessage',
+    {
+      defaultMessage:
+        'Alert for monitor {monitorName} with url {monitorUrl} from {observerLocation} has recovered',
+      values: {
+        monitorName: '{{context.monitorName}}',
+        monitorUrl: '{{{context.monitorUrl}}}',
+        observerLocation: '{{context.observerLocation}}',
+      },
+    }
+  ),
+  name: i18n.translate('xpack.synthetics.alerts.monitorStatus.clientName', {
+    defaultMessage: 'Uptime monitor status',
+  }),
+  description: i18n.translate('xpack.synthetics.alerts.monitorStatus.description', {
+    defaultMessage: 'Alert when a monitor is down or an availability threshold is breached.',
+  }),
+};
+
+export const TlsTranslations = {
+  defaultActionMessage: i18n.translate('xpack.synthetics.alerts.tls.defaultActionMessage', {
+    defaultMessage: `Detected TLS certificate {commonName} from issuer {issuer} is {status}. Certificate {summary}`,
+    values: {
+      commonName: '{{context.commonName}}',
+      issuer: '{{context.issuer}}',
+      summary: '{{context.summary}}',
+      status: '{{context.status}}',
+    },
+  }),
+  defaultRecoveryMessage: i18n.translate('xpack.synthetics.alerts.tls.defaultRecoveryMessage', {
+    defaultMessage: `Alert for TLS certificate {commonName} from issuer {issuer} has recovered`,
```

Comment:
```suggestion
    defaultMessage: `The alert for TLS certificate {commonName} from issuer {issuer} is no longer active`,
```
Feel free to ignore my changes from "has recovered" to "is no longer active". Since the subject is "the alert", it sounds strange to me that an alert recovers, so suggesting an alternative if "the alert" has to remain the main subject. This comment also applies to similar text starting with "the alert".